init-service/src/main/resources/application-local.yml

```diff
@@ -17,6 +17,9 @@ logging:
               client:
                 ExchangeFunctions: TRACE
 
+auth:
```

Comment:
All mts related properties are now mts.some.thing...
I think that mts.authorization.backend-scope fits (but just a suggestion).

Reply:
Following our conversation, this will require to add it to the config validation. Will be done in a separate pr.